source/DSCResources/xSccmPreReqs/xSccmPreReqs.schema.psm1

```diff
@@ -0,0 +1,372 @@
+
+    .PARAMETER AdkWinPeSetupPath
+        Specifies the path and filename to the ADK WinPE Setup.
+
+    .PARAMETER MdtMsiPath
+        Specifies the path and filename to the MDT Setup.
+
+    .PARAMETER InstallWindowsFeatures
+        Specifiy to Install Windows Features needed for the SCCM install.
+
+    .PARAMETER WindowsFeatureSource
+        Specifies the source that will be used to install windows features if the files are not present in the local
+        side-by-side store.
+
+    .PARAMETER SccmRole
+        Specify the SCCM Roles that will be on the server.
+
+    .PARAMETER AddWindowsFirewallRule
+        Specify whether to add the Windows Firewall Rules needed for the install.
+
+    .PARAMETER FirewallProfile
+        Specify the Windows Firewall profile for the rules to be added.
+
+    .PARAMETER FirewallTcpLocalPort
+        Specify the TCP ports to be added to the windows firewall as allowed.
+
+    .PARAMETER FirewallUdpLocalPort
+        Specify the UDP ports to be added to the windows firewall as allowed.
+
+    .PARAMETER LocalAdministrators
+        Specifies the accounts and/or groups you want to add to the local administrators group.
+
+    .PARAMETER NoSmsOnDrives
+        Specifies the drive letters of the drive you don't want SCCM to install on.
+
+    .PARAMETER DomainCredential
+        Specifies credentials that have domain read permissions to add domain users or groups to the local
+        administrators group.
+
+    .PARAMETER AdkProductName
+        Specifies the Product Name for ADK.
+        Default: Windows Assessment and Deployment Kit - Windows 10
+
+    .PARAMETER AdkProductID
+        Specifies the Product ID for ADK.
+        Default: fb450356-9879-4b2e-8dc9-282709286661
+
+    .PARAMETER AdkWinPeProductName
+        Specifies the Product Name for  ADK WinPE.
+        Default: Windows Assessment and Deployment Kit Windows Preinstallation Environment Add-ons - Windows 10
+
+    .PARAMETER AdkWinPeProductID
+        Specifies the Product ID for ADK WinPE.
+        Default: d8369a05-1f4a-4735-9558-6e131201b1a2
+
+    .PARAMETER AdkInstallPath
+        Specifies the path to install ADK and ADK WinPE.
+        Default: C:\Program Files (x86)\Windows Kits\10
+
+    .PARAMETER MdtProductName
+        Specifies the Product Name for MDT.
+        Default: Microsoft Deployment Toolkit (6.3.8456.1000)
+
+    .PARAMETER MdtProductID
+        Specifies the Product ID for MDT.
+        Default: 2E6CD7B9-9D00-4B04-882F-E6971BC9A763
+
+    .PARAMETER MdtInstallPath
+        Specifies the path to install MDT.
+        Default: C:\Program Files\Microsoft Deployment Toolkit
+
+    .NOTES
+        SCCM Roles based one the following documentation:
+        https://docs.microsoft.com/en-us/mem/configmgr/core/plan-design/configs/site-and-site-system-prerequisites
+#>
+Configuration xSCCMPreReqs
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter()]
+        [Boolean]
+        $InstallADK = $true,
```

Comment:
Fix casing for the variable throughout. $InstallAdk

Reply:
fixed.